js/renovation/viz/core/common/tooltip_utils.ts

```diff
@@ -285,16 +290,14 @@ export function getCloudAngle(
 }
 
 export function prepareData(
-  data: TooltipData, color: string,
-  border: InitialBorder,
-  font: {
-    color: string; family: string; opacity: number; size: number; weight: number;
-  },
+  data?: TooltipData, color?: string,
+  border?: InitialBorder,
+  font?: Font,
   customizeTooltip?: CustomizeTooltipFn,
 ): CustomizedOptions {
   let customize = {} as CustomizedOptions;
 
-  if (isFunction(customizeTooltip)) {
+  if (customizeTooltip && isFunction(customizeTooltip)) {
```

Comment:
I think the additional condition is excess. The `isFunction` function enough.

Reply:
Ok

Reply:
linter says it's a bad idea to remove this check because customizeTooltip can be undefined (ESLint doesn't know what isFunction checking)

Reply:
https://trello.com/c/oSnG9nzt/1360-remove-undefined-operator-from-tooltip-props